commonmark_extensions/tests.py

```diff
@@ -141,6 +143,7 @@ def test_break2(self):
 </tr>
 </tbody>
 </table>
+
```

Comment:
This differs slightly from the spec, but doesn't actually change the visual output of the html (as blank lines aren't treated specially).

If you are bothered by this, let me know and I can try work out why the new line appears and whether we can avoid it.